requirements.txt

```diff
@@ -11,3 +11,4 @@ numpy ~= 1.21.4  # Sync with the numpy version used in TF
 black==22.3.0
 isort==5.10.1
 flake8==4.0.1
+flynt==0.76
```

Comment:
Rather than added this dependency I would recommend a one-time update of the codebase.

Reply:
Sure, no problems, I can remove the dependency. I added it because I thought it could go well in there as it shares the same "goal" as of the others such `isort`.

Reply:
Done.